pyproject.toml

```diff
@@ -533,6 +533,7 @@ alibaba = [
   "oss2>=2.14.0",
 ]
 amazon = [
+  "PyAthena>=3.0.10",
```

Comment:
Is this a related change?

Reply:
It came in with one of the merge conflict resolutions.